packages/terra-search-field/src/SearchField.scss

```diff
@@ -4,24 +4,32 @@
 
     .input {
       appearance: none;
+      border-bottom-left-radius: var(--terra-search-field-border-bottom-left-radius, 0.25em);
       border-bottom-right-radius: 0;
       border-right: 0;
+      border-top-left-radius: var(--terra-search-field-border-top-left-radius, 0.25em);
```

Comment:
To avoid collisions down the road we should match the prefix used on the margin properties below: 

`--terra-search-field-input`